### DISCUSSION TITLE
How to make svg files previewable

### DISCUSSION THREAD

**Discussion by onweru** [sveltia/sveltia-cms#177] [Q&A]
When the value of an image field is svg, it's not previewable. I'm I missing something?

**Comment by kyoshino** [comment]
It should just work. Will check why the preview is not available.

**Reply by kyoshino** [reply]
I have tested with several SVG images and don’t see any issues. Perhaps your SVG is corrupt, e.g. missing `viewbox`?

**Reply by onweru** [reply]
Let me check.
